internal/pkg/cli/pipeline_init.go

```diff
@@ -450,10 +446,18 @@ func BuildPipelineInitCmd() *cobra.Command {
     --environments "stage,prod" \
     --deploy`,
 		PreRunE: runCmdE(func(cmd *cobra.Command, args []string) error {
+			err := opts.Validate()
+			if err != nil {
+				return err
+			}
```

Comment:
nit: you could consolidate these two lines with
```go
if err := opts.Validate(); err !=nil {
   return err
}
```

Reply:
:+1: